scripts/azure-pipelines/build_macos.bash

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -ex
+#
+# Copyright 2005-2019 The Mumble Developers. All rights reserved.
```

Comment:
This is a new file. Copyright should start in 2019.

Reply:
That's not the case for other scripts, the Travis CI ones have `2005-2019`.

Reply:
Do we have it for all files? Do we make it a generic statement?

Otherwise, nothing was done in the years before, so it'd be wrong to claim so. The copyright years are an indication of when the content was created - to know when potentially it expires.

Reply:
It's considered a generic statement, however I agree.